quickwit/quickwit-serve/src/otlp_api/rest_handler.rs

```diff
@@ -47,6 +48,7 @@ pub(crate) fn otlp_ingest_api_handlers(
         .or(otlp_default_traces_handler(otlp_traces_service.clone()))
         .or(otlp_logs_handler(otlp_logs_service))
         .or(otlp_ingest_traces_handler(otlp_traces_service))
+        .recover(recover_fn)
```

Comment:
not related to this PR: this group of routes isn't present in the openapi definition

Reply:
added routes to openapi..